src/components/block-tunes/block-tune-delete.ts

```diff
@@ -6,6 +6,7 @@
  */
 import { API, BlockTune } from '../../../types';
 import $ from '../dom';
+import Popover, { PopoverItem } from '../utils/popover';
```

Comment:
```suggestion
import { PopoverItem } from '../utils/popover';
```

Reply:
can't remove this import. Popover.CSS is used for animation.